iahx/templates/custom/result-inc-source.html

```diff
@@ -1,11 +1,11 @@
 <!-- source -->
 {% if doc.fo %}
-    <div>
+    <div class="line source">
```

Comment:
Me refiro trecho que colocar: Brazilian journal of biology, fev 2009, Volume 69 N 1 e etc....